\doxysection{pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Unrewindable\+Body\+Error Class Reference}
\hypertarget{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1UnrewindableBodyError}{}\label{classpip_1_1__vendor_1_1urllib3_1_1exceptions_1_1UnrewindableBodyError}\index{pip.\_vendor.urllib3.exceptions.UnrewindableBodyError@{pip.\_vendor.urllib3.exceptions.UnrewindableBodyError}}


Inheritance diagram for pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Unrewindable\+Body\+Error\+:
% FIG 0


Collaboration diagram for pip.\+\_\+vendor.\+urllib3.\+exceptions.\+Unrewindable\+Body\+Error\+:
% FIG 1


\doxysubsection{Detailed Description}
\begin{DoxyVerb}urllib3 encountered an error when trying to rewind a body\end{DoxyVerb}
 

Definition at line \mbox{\hyperlink{pip_2__vendor_2urllib3_2exceptions_8py_source_l00320}{320}} of file \mbox{\hyperlink{pip_2__vendor_2urllib3_2exceptions_8py_source}{exceptions.\+py}}.



The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
/home/jam/\+Research/\+IRES-\/2025/dev/src/llm-\/scripts/testing/hypothesis-\/testing/hyp-\/env/lib/python3.\+12/site-\/packages/pip/\+\_\+vendor/urllib3/exceptions.\+py\end{DoxyCompactItemize}
